docs/source/main_classes/trainer.rst

```diff
@@ -113,7 +113,117 @@ Toward Training Trillion Parameter Models, by Samyam Rajbhandari, Jeff Rasley, O
 
 This provided support is new and experimental as of this writing.
 
-You will need at least 2 GPUs to benefit from these features.
+Installation Notes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As of this writing, both, FairScale and Deepspeed, require compilation of CUDA C++ code, before they can be used.
+
+While all installation issues should be dealt with through the corresponding github Issues of FairScale and DeepSpeed,
```

Comment:
```suggestion
While all installation issues should be dealt with through the corresponding GitHub Issues of FairScale and DeepSpeed,
```

Reply:
+ links to issues of FairScale and DeepSpeed (under their respective names) here?